gnomad_hail/resources/import_resources.py

```diff
@@ -0,0 +1,83 @@
+from inspect import getmembers
+from typing import Dict, Tuple, Optional
+from gnomad_hail.resources.resource_utils import BaseResource, BaseVersionedResource
+import gnomad_hail.resources.grch37 as grch37
+import gnomad_hail.resources.grch38 as grch38
+import argparse
+
+
+# Generate a dictionary of resource available for import for a given genome build
+def get_module_importable_resources(module, prefix: Optional[str] = None) -> Dict[str, Tuple[str, BaseResource]]:
```

Comment:
I'm envisioning importing this function to import resources in other projects, e.g. gnomad_qc/v3 and applying it to import project-specific resources.

Reply:
Why would we use this function to import resources over `from gnomad_hail.resources import grch37`?

Reply:
This would be used to create import scripts for project-specific resources. An example is VQSR that is generated for each project as VCF using GATK and then needs importing in hail. But this file isn't public as it contains sites from unreleasable samples. Samples metadata is another example. So by having this generic functions, we can make simple project-specific import scripts that are only a few lines long

Reply:
Oh, ok. I think I misunderstood. I thought you were saying use this function to import resources from the GRCh37/38 resources modules here.

Reply:
Right, I can see how my comment was ambiguous :)